cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/FileSystemUtil.java

```diff
@@ -72,6 +75,18 @@ public static FileSystem getOrCreateFileSystem(URI fileSystemUri) {
     return configFileSystem;
   }
 
+  /**
+   * @param properties path to a properties file
+   * @param output string to be written in the properties
+   * @throws FileNotFoundException
+   */
+  public static void writeOutputToFile(Path properties, String output) throws FileNotFoundException {
```

Comment:
Please replace with: https://www.baeldung.com/java-properties
6.1. Store to Properties Files